Maskbook/Scene/MainTab/MainTabBarController.swift

```diff
@@ -81,14 +81,6 @@ class MainTabBarController: UITabBarController {
 }
 
 extension MainTabBarController {
-    override var childForStatusBarStyle: UIViewController? {
```

Comment:
Why delete this？

Reply:
This method has moved into `NavigationController`.